enterprise_access/apps/api/v1/views/subsidy_access_policy.py

```diff
@@ -966,25 +1024,58 @@ def get_group_member_data_with_aggregates(self, request, uuid):
             policy.uuid
         )
 
+        # If `sort_by_enrollment_count` is true, then we need to fetch all the group members records and do the sorting
+        # ourselves since platform is unaware of enrollment data numbers.
+        request_page = None
```

Comment:
nit: a lot of the code below would read much cleaner if you renamed this as `page_requested_by_client` or something.